pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
   'absl-py',
   'glyphsLib',
   'gflanguages>=0.4.0',
+  'gfsubsets>=2024.02.05',
```

Comment:
is this a typo, or do dates actually work?

Reply:
Yeah, they work fine: major version 2024, minor version 2, patch version 5.